### PR TITLE
feat(spotify-player): add quicklink to add playing song to a specific playlist

### PR DESCRIPTION
**Pull request by erics118**
## Description

adds a command to the `Add Playing Song to Playlist` command to create a quicklink so that users can easily add a song to a specific playlist, rather than having to go through the List view every single time.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/spotify-player/package.json [hunk 1]
```diff
@@ -30,6 +30,15 @@
...
   ],
+  "debug": {
+    "reloadShortcut": {
+      "key": "r",
+      "modifiers": [
+        "command",
+        "shift"
+      ]
+    }
+  },
```
Comment (thomaslombart) [inline extensions/spotify-player/package.json:41]: What is this used for?

Comment (thomaslombart) [inline extensions/spotify-player/package.json:41]: Why is this needed?

Comment (erics118) [reply extensions/spotify-player/package.json:41]: This changes the shortcut to reload the entire extension to cmd-shift-r to prevent conflicts, as cmd-r is already used in some places by the extension to . It's not documented in the [docs](https://developers.raycast.com), but it does appear in the [schema](https://www.raycast.com/schemas/extension.json).

**Review by thomaslombart — COMMENTED** [review]
I'm getting duplicate songs when using the quicklink action. Are you sure this has been fixed?

**Comment by erics118** [comment]
It seems to be only adding songs once for me. The `useEffect` should've made sure that the logic is only added once.

**Comment by thomaslombart** [comment]
Not for me 😕

https://github.com/user-attachments/assets/e9c43c41-885d-43f2-8a41-3c2e1f077906

**Comment by erics118** [comment]
Weird, not sure what could be causing this.

**Comment by erics118** [comment]
Reopen

**Comment by erics118** [comment]
@thomaslombart Could you reopen this PR and if possible, take a look as to what might be causing this? I really don't know what could be the bug. Thanks!

**Comment by pernielsentikaer** [comment]
I just gave it a spin and for me it also just adds the song once 🤔

**Comment by erics118** [comment]
That's pretty weird. Could it be that before, an older version of the branch was used, without the proper useEffect fix?

**Comment by thomaslombart** [comment]
I've tested it again and I can't reproduce the error anymore, which is weird.

**Review by thomaslombart — APPROVED** [review]

**Comment by thomaslombart** [comment]
Let me know if it's good for you @pernielsentikaer
